src/elements/content-preview/ContentPreview.js

```diff
@@ -62,7 +63,6 @@ type Props = {
     collection: Array<string | BoxItem>,
     contentOpenWithProps: ContentOpenWithProps,
     contentSidebarProps: ContentSidebarProps,
-    contentSidebarRef: React.Ref<any>,
```

Comment:
We'll definitely want to let our partners team know about this change.

Reply:
Sure thing! Can you point me in the right direction so I can let them know?